src/sentry/api/event_search.py

```diff
@@ -1174,7 +1174,7 @@ def has_default(self, params):
         "name": "user_misery",
         "args": [NumberRange("satisfaction", 0, None)],
         "calculated_args": [{"name": "tolerated", "fn": lambda args: args["satisfaction"] * 4.0}],
-        "transform": u"uniqIf(user, duration > {tolerated:g})",
+        "transform": u"uniqIf(user, greater(duration, {tolerated:g}))",
```

Comment:
@evanh I am not very familiar with this part of the code base. Is this goingn to impact the aggregation field for snuba only or others ?